enclave/core/arena.c

```diff
@@ -89,13 +94,16 @@ void* oe_arena_calloc(size_t num, size_t size)
 
 void oe_arena_free_all()
 {
-    _arena.used = 0;
+    oe_shared_memory_arena_t* arena = _get_arena();
+    arena->used = 0;
 }
 
 // Free the arena in the current thread.
 void oe_teardown_arena()
```

Comment:
Is any of this stuff called in OP-TEE? We should probably move arena.c and arena.h to enclave/core/sgx

Reply:
Good idea. Done.